tensorflow/core/kernels/list_kernels.cc

```diff
@@ -336,6 +336,13 @@ REGISTER_KERNEL_BUILDER(Name("TensorListPushBack").Device(DEVICE_GPU),
 
 #endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 
+#if TENSORFLOW_USE_DIRECTML
```

Comment:
Inconsistent with the one below. It should be `#ifdef TENSORFLOW_USE_DIRECTML`

Reply:
Changed all #ifdef to #if to keep consistent with #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM

---
In reply to: [514683885](https://github.com/microsoft/tensorflow-directml/pull/78#discussion_r514683885) [](ancestors = 514683885)

Reply:
That's not being consistent though. We use `#ifdef TENSORFLOW_USE_DIRECTML` everywhere (in the same manner that sycl uses `#ifdef TENSORFLOW_USE_SYCL`). CUDA can't use ifdef here because there are 2 conditions.